### ISSUE TITLE
GStreamer1.24 issue (Arch Linux, Ubuntu-24.04)

### ISSUE THREAD

**Issue by fduncanh** [FDH2/UxPlay#288]
uxplay-1.68.2 is broken by a change (involving pause/resume) in GStreamer-1.24, used in latest Arch Linux, and now
Ubuntu-24.04

solution:  
(1) downgrade gstreamer to 1.22.x if possible
or
(2) use uxplay-1.68.3 from this site.   (now in ArchLinux AUR, hopefully soon in Ubuntu-24.04)

**Comment by Utopiah** [comment]
Is this related to 

```
Required gstreamer plugin 'libav' not found
uxplay: ./renderers/audio_renderer_gstreamer.c:83: audio_renderer_init: Assertion `check_plugins ()' failed.
Aborted (core dumped)
```

on Ubuntu 22.04.4 LTS too?

**Comment by fduncanh** [comment]
> Is this related to
>
>Required gstreamer plugin 'libav' not found
uxplay: ./renderers/audio_renderer_gstreamer.c:83: audio_renderer_init: Assertion `check_plugins ()' failed.
Aborted (core dumped)
>
>for version 1.46-1 on Ubuntu 22.04.4 LTS too?

No, not related.

**Comment by fduncanh** [comment]
this is now fixed in uxplay-1.70
